self-api/src/main/java/com/selfxdsd/api/Repo.java

```diff
@@ -81,4 +81,10 @@
      * @return Webhooks.
      */
     Webhooks webhooks();
+
+    /**
+     * Star this repo.
+     * @return Stars
+     */
+    Stars star();
```

Comment:
@ak2703 Can you name this method ``stars()`` (plural)? That's the correct way, since it's the entry point of a Repo's Stars. 

Besides, as it is now, starring a repo will look weird: ``repo.star().star()``.